velox/functions/prestosql/aggregates/tests/MinMaxByAggregationTest.cpp

```diff
@@ -115,7 +122,7 @@ std::vector<TestParam> getTestParams() {
 
 class MinMaxByAggregationTestBase : public AggregationTestBase {
  protected:
-  MinMaxByAggregationTestBase() : numValues_(6) {}
+  MinMaxByAggregationTestBase() : numValues_(7) {}
```

Comment:
just curious about the change of numValues_ as to why it was moved to 7.  I wonder if the tests for date only failed after the move to 7 values.

Reply:
Thanks for pointing it out, upon review I see this change wasn't needed, but the tests for date failed without this change. Would it be fine if numValues_ remains 7 or would you suggest reverting it back to 6? It should be fine either way.

Reply:
Thanks for checking, should be fine if we leave it as 7.
Only reason I asked was that I was just curious if there was more to it, and if so would have suggested to document it.

Reply:
> upon review I see this change wasn't needed, but the tests for date failed without this change

What is the failure?

Reply:
Sorry for the confusion, I meant the min_by, max_by tests for date are not failing because of this change. The failure is because of the incorrect initialization as updated in the PR description.

Reply:
@pramodsatya I'm trying to understand why we need numValues_ to be 7 and not 6? Can it be 6? If so, let's revert the change to 7 to avoid confusion.

Reply:
Thank you, @pramodsatya .